internal/pkg/deploy/cloudformation/lb_fargate_app.go

```diff
@@ -28,25 +49,60 @@ func NewLBFargateStack(in *deploy.CreateLBFargateAppInput) *LBFargateStackConfig
 
 // StackName returns the name of the stack.
 func (c *LBFargateStackConfig) StackName() string {
-	return ""
+	const maxLen = 128
```

Comment:
QQ: Is this a constraint that CFN defines?

Reply:
> It must start with an alphabetic character and can't be longer than 128 characters.   

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console-create-stack-parameters.html

Yup, added a comment with the link in the code